src/__tests__/reduxForm.spec.js

```diff
@@ -185,25 +185,25 @@ const describeReduxForm = (name, structure, combineReducers, expect) => {
       const spy = createSpy()
       const { dispatch } = propChecker({}, spy, {
         validate: values => {
-          const animal = getIn(values, 'animal')
-          return animal && animal.length > 5 ? { animal: 'Too long' } : {}
+          const foo = getIn(values, 'foo')
+          return foo && foo.length > 5 ? { foo: 'Too long' } : {}
```

Comment:
You got something against animals? 🐘 🐶 🐱 🐴 

I'm just kidding. Just amused by arbitrary change. 😃 Don't change it back.

![animals](https://cloud.githubusercontent.com/assets/4396759/15302864/6ac8f05a-1bb5-11e6-80d8-81d768ae571b.jpg)